RevenueCatUI/CustomerCenter/Data/PurchaseInformation.swift

```diff
@@ -236,3 +220,36 @@ fileprivate extension String {
     }
 
 }
+
+protocol Transaction {
```

Comment:
This is probably a bad name, there are too many things named `Transaction` already